src/rocdecode/vaapi/vaapi_videodecoder.cpp

```diff
@@ -301,6 +304,30 @@ rocDecStatus VaapiVideoDecoder::SubmitDecode(RocdecPicParams *pPicParams) {
             break;
         }
 
+        case rocDecVideoCodec_AV1: {
+            pPicParams->pic_params.av1.current_frame = curr_surface_id;
```

Comment:
Also add surface ID for current_display_picture and anchor_frames_list as well, for future proof.